Source/RapyutaSimulationPlugins/Public/Robots/RRRobotROS2Interface.h

```diff
@@ -298,10 +297,9 @@ class RAPYUTASIMULATIONPLUGINS_API URRRobotROS2Interface : public UObject
     void MakeServiceRequest(const FName& InServiceName, const TServiceRequest& InRequest)
     {
         // Create and update request
-        auto* clienPtr = ServiceClientList.Find(InServiceName);
-        if (clienPtr)
+        if (auto* clientPtr = ServiceClientList.Find(InServiceName))
         {
-            UROS2ServiceClient* client = *clienPtr;
+            UROS2ServiceClient* client = *clientPtr;

```

Comment:
Do you need to define new var here?

Reply:
This change only renamed an existing variable that had a typo `clienPtr` to [clientPtr](url).

Reply:
My question is do you need to define `client`? can't you directly use clientPtr?

Reply:
I think we need it, @Tadinu could you help to explain this?

Reply:
Then I will change it to 
```
if (auto* client = ServiceClientList.FindRef(InServiceName))
{
}
```